src/layouts/LinkReport/components/LinkReportModal/LinkReportModal.tsx

```diff
@@ -0,0 +1,549 @@
+                                    </Text>
+                                  )}
+                                  <Text
+                                    color={colors.base.content.strong}
+                                    textStyle="body-2"
+                                    paddingBottom=".75rem"
+                                    fontFamily={typography.fontFamilies.inter}
+                                  >
+                                    {link.linkToAsset
+                                      ? link.linkToAsset
+                                      : "No URL linked"}
+                                  </Text>
+                                  <HStack>
+                                    <svg
+                                      xmlns="http://www.w3.org/2000/svg"
+                                      width="16"
+                                      height="16"
+                                      viewBox="0 0 16 16"
+                                      fill="none"
+                                    >
+                                      <path
+                                        d="M8.00004 1.3335C4.32404 1.3335 1.33337 4.32416 1.33337 8.00016C1.33337 11.6762 4.32404 14.6668 8.00004 14.6668C11.676 14.6668 14.6667 11.6762 14.6667 8.00016C14.6667 4.32416 11.676 1.3335 8.00004 1.3335ZM8.66671 11.3335H7.33337V7.3335H8.66671V11.3335ZM8.66671 6.00016H7.33337V4.66683H8.66671V6.00016Z"
+                                        fill="#C03434"
+                                      />
+                                    </svg>
+                                    <Text
+                                      color={colors.utility.feedback.critical}
+                                      textStyle="caption-1"
+                                      fontFamily={typography.fontFamilies.inter}
+                                    >
+                                      {getSuggestion(link)}
+                                    </Text>
+                                  </HStack>
+                                </VStack>
+                              </Td>
+                            </Tr>
+                          )
+                        })}
+                    </Tbody>
+                  )}
+
+                  {isBrokenLinksLoading && (
+                    <Tbody>
+                      <Tr borderTop="1px" borderColor="base.divider.medium">
+                        <Td colSpan={3} border="0px">
+                          <LinkReportModalLoading />
+                        </Td>
+                      </Tr>
+                    </Tbody>
+                  )}
+                </Table>
+              </TableContainer>
+            </>
+          )}
+        </VStack>
+      </GridItem>
+    </Grid>
+  )
+}
+
+type DetailedNonPermaLinkError = NonPermalinkError & {
+  detailedType: string
+}
+
+const isEmailError = (error: NonPermalinkError): boolean => {
+  return (
```

Comment:
see if you can use the email regex that is already defined in the codebase

Reply:
Doesn't seem to work for me, maybe I am doing something wrong?